addons/binding/org.openhab.binding.omnilink/ESH-INF/thing/unit.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- These are all objects that are types of Units -->
+	
+	<thing-type id="unit">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="controller" />
+        </supported-bridge-type-refs>
+        <label>Unit</label>
+        <description>A basic unit object</description>
+        <channels>
+            <channel id="status" typeId="status" />
+        </channels>
+        <properties>
+            <property name="name" />
+            <property name="area" />
+            <property name="room" />
```

Comment:
So names and areas are properties of most objects,  I am adding the room of a unit as well, any objections?